Project-Centric build cmake configure error - could not find Packaging_windows.cmake

Following this guide:
https://docs.o3de.org/docs/welcome-guide/get-started/project-config/creating-projects-using-cli/

clone o3de commit d8bd6ef407e88d5c4029cb9607d698c4409a3bab (main branch at time of writing)

Steps:
```
cd c:/code/o3de
./scripts/o3de.bat create-project --project-path "c:\code\SomeProject2"
./scripts/o3de.bat register -p "c:\code\SomeProject2"
cd ../SomeProject2
cmake.exe -B build -S . -G "Visual Studio 16 2019" -DLY_3RDPARTY_PATH="c:\o3de-3p\3rdParty" -DLY_UNITY_BUILD=ON
```
cmake version is 3.19.1

I get the following:
```-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.19042.
-- The C compiler identification is MSVC 19.28.29337.0
-- The CXX compiler identification is MSVC 19.28.29337.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- o3de_manifest_name: alt
-- default_engines_folder: C:/Users/alt/.o3de/Engines
-- default_projects_folder: C:/Users/alt/.o3de/Projects
-- default_gems_folder: C:/Users/alt/.o3de/Gems
-- default_templates_folder: C:/Users/alt/.o3de/Templates
-- default_restricted_folder: C:/Users/alt/.o3de/Restricted
-- Found engine: 'o3de' at C:/code/o3de
-- O3DE Project Name: SomeProject2
-- O3DE Project Path: C:/code/SomeProject2
-- O3DE Project Restricted Path: C:/Users/alt/.o3de/Projects/Restricted
-- O3DE Engine Name: o3de
-- O3DE Engine Path: C:/code/o3de
-- O3DE Engine Restricted Path: C:/Users/alt/.o3de/Restricted
-- Using Windows target SDK 10.0.18362.0
-- Using package C:/o3de-3p/3rdParty/packages/qt-5.15.2-rev2-windows
-- Using package C:/code/o3de/python/runtime/python-3.7.10-rev1-windows
-- Using Python 3.7.10 at C:/code/o3de/python/runtime/python-3.7.10-rev1-windows/python/python.exe
-- Python: Getting/Checking packages listed in C:/code/o3de/python/requirements.txt
-- Python: Getting/Checking packages listed in C:/code/o3de/python/requirements.txt - New packages were installed
-- Python: linking C:/code/o3de/Tools/LyTestTools into python site-packages...
-- Python: linking C:/code/o3de/Tools/RemoteConsole/ly_remote_console into python site-packages...
-- Python: linking C:/code/o3de/AutomatedTesting/Gem/PythonTests/EditorPythonTestTools into python site-packages...
-- Using package C:/o3de-3p/3rdParty/packages/expat-2.1.0-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/lz4-r128-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/md5-2.0-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/tiff-4.2.0.14-windows
-- Using package C:/o3de-3p/3rdParty/packages/zlib-1.2.8-rev2-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/zstd-1.35-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/RapidJSON-1.1.0-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/Lua-5.3.5-rev5-windows
-- Using package C:/o3de-3p/3rdParty/packages/RapidXML-1.13-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/cityhash-1.1-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/googletest-1.8.1-rev4-windows
-- Using package C:/o3de-3p/3rdParty/packages/googlebenchmark-1.5.0-rev2-windows
-- Using package C:/o3de-3p/3rdParty/packages/SQLite-3.32.2-rev3-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/OpenSSL-1.1.1b-rev2-windows
-- Using package C:/o3de-3p/3rdParty/packages/squish-ccr-20150601-rev3-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/AWSNativeSDK-1.7.167-rev3-windows
-- Using package C:/o3de-3p/3rdParty/packages/assimp-5.0.1-rev7-multiplatform
-- Python: linking C:/o3de-3p/3rdParty/packages/assimp-5.0.1-rev7-multiplatform/assimp/port/PyAssimp into python site-packages...
-- Using package C:/o3de-3p/3rdParty/packages/xxhash-0.7.4-rev1-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/Crashpad-0.8.0-rev1-windows
-- Using package C:/o3de-3p/3rdParty/packages/pybind11-2.4.3-rev2-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/pyside2-qt-5.15.1-rev2-windows
-- Using package C:/o3de-3p/3rdParty/packages/etc2comp-9cd0f9cae0-rev1-windows
-- Using package C:/o3de-3p/3rdParty/packages/PVRTexTool-4.24.0-rev4-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/ilmbase-2.3.0-rev4-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/ASTCEncoder-2017_11_14-rev2-multiplatform
-- Downloading package into C:/o3de-3p/3rdParty/packages/mcpp-2.7.2_az.1-rev1-windows
'C:/o3de-3p/3rdParty/CMake/3.19.1/bin/cmake.exe' '-E' 'tar' 'xf' 'C:/o3de-3p/3rdParty/downloaded_packages/mcpp-2.7.2_az.1-rev1-windows.tar.xz'
-- Installed And Validated package at C:/o3de-3p/3rdParty/packages/mcpp-2.7.2_az.1-rev1-windows - OK
-- Using package C:/o3de-3p/3rdParty/packages/mcpp-2.7.2_az.1-rev1-windows
-- Downloading package into C:/o3de-3p/3rdParty/packages/DirectXShaderCompiler-1.6.2104-o3de-rev1-windows
'C:/o3de-3p/3rdParty/CMake/3.19.1/bin/cmake.exe' '-E' 'tar' 'xf' 'C:/o3de-3p/3rdParty/downloaded_packages/DirectXShaderCompiler-1.6.2104-o3de-rev1-windows.tar.xz'
-- Installed And Validated package at C:/o3de-3p/3rdParty/packages/DirectXShaderCompiler-1.6.2104-o3de-rev1-windows - OK
-- Using package C:/o3de-3p/3rdParty/packages/DirectXShaderCompiler-1.6.2104-o3de-rev1-windows
-- Using package C:/o3de-3p/3rdParty/packages/SPIRVCross-2020.04.20-rev1-multiplatform
-- Downloading package into C:/o3de-3p/3rdParty/packages/azslc-1.7.21-rev1-multiplatform
'C:/o3de-3p/3rdParty/CMake/3.19.1/bin/cmake.exe' '-E' 'tar' 'xf' 'C:/o3de-3p/3rdParty/downloaded_packages/azslc-1.7.21-rev1-multiplatform.tar.xz'
-- Installed And Validated package at C:/o3de-3p/3rdParty/packages/azslc-1.7.21-rev1-multiplatform - OK
-- Using package C:/o3de-3p/3rdParty/packages/azslc-1.7.21-rev1-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/lux_core-2.2-rev5-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/d3dx12-headers-rev1-windows
-- Using package C:/o3de-3p/3rdParty/packages/freetype-2.10.4.14-windows
-- Python: Getting/Checking packages listed in C:/code/o3de/Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/requirements.txt
-- Python: Getting/Checking packages listed in C:/code/o3de/Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/requirements.txt - New packages were installed
-- Wwise SDK version 2021.1.1.7601 was not found.
-- ** Update the LY_WWISE_INSTALL_PATH cache variable if you intend to use Wwise.
-- Using package C:/o3de-3p/3rdParty/packages/Blast-1.1.7-rev1-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/PhysX-4.1.0.25992954-rev1-windows
-- Found OpenGL: opengl32
-- Using package C:/o3de-3p/3rdParty/packages/libsamplerate-0.2.1-rev2-windows
-- Using package C:/o3de-3p/3rdParty/packages/NvCloth-1.1.6-rev1-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/poly2tri-0.3.3-rev2-multiplatform
-- Using package C:/o3de-3p/3rdParty/packages/v-hacd-2.0-rev1-multiplatform
-- Python: linking C:/o3de-3p/3rdParty/packages/pyside2-qt-5.15.1-rev2-windows/pyside2 into python site-packages...
-- Using package C:/o3de-3p/3rdParty/packages/mikkelsen-1.0.0.4-windows
-- Using package C:/o3de-3p/3rdParty/packages/ISPCTexComp-2021.3-rev1-windows
-- Using package C:/o3de-3p/3rdParty/packages/openimageio-2.1.16.0-rev2-windows
-- Using package C:/o3de-3p/3rdParty/packages/civetweb-1.8-rev1-windows
-- Using package C:/o3de-3p/3rdParty/packages/OpenMesh-8.1-rev1-windows
CMake Error at C:/code/o3de/cmake/Packaging.cmake:44 (include):
  include could not find load file:

    C:/code/SomeProject2/cmake/Platform/Windows/Packaging_windows.cmake
Call Stack (most recent call first):
  C:/code/o3de/CMakeLists.txt:153 (include)


-- Configuring incomplete, errors occurred!
See also "C:/code/SomeProject2/build/CMakeFiles/CMakeOutput.log".

```

Seems like it's trying to reference
`C:/code/SomeProject2/cmake/Platform/Windows/Packaging_windows.cmake` 
when it should be referenceing
`C:/code/o3de/cmake/Platform/Windows/Packaging_windows.cmake`

--- a/cmake/Packaging.cmake
+++ b/cmake/Packaging.cmake
@@ -29,15 +29,15 @@ string(TOLOWER ${PROJECT_NAME} _project_name_lower)
 set(CPACK_PACKAGE_FILE_NAME "${_project_name_lower}_${LY_VERSION_STRING}_installer")
 
 set(DEFAULT_LICENSE_NAME "Apache-2.0")
-set(DEFAULT_LICENSE_FILE "${CMAKE_SOURCE_DIR}/LICENSE.txt")
+set(DEFAULT_LICENSE_FILE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE.txt")
 
 set(CPACK_RESOURCE_FILE_LICENSE ${DEFAULT_LICENSE_FILE})
 set(CPACK_LICENSE_URL ${LY_INSTALLER_LICENSE_URL})
 
 set(CPACK_PACKAGE_INSTALL_DIRECTORY "${CPACK_PACKAGE_VENDOR}/${CPACK_PACKAGE_VERSION}")
 
-# CMAKE_SOURCE_DIR doesn't equate to anything during execution of pre/post build scripts
-set(CPACK_SOURCE_DIR ${CMAKE_SOURCE_DIR}/cmake)
+# neither of the SOURCE_DIR variables equate to anything during execution of pre/post build scripts
+set(CPACK_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 # attempt to apply platform specific settings
 ly_get_absolute_pal_filename(pal_dir ${CPACK_SOURCE_DIR}/Platform/${PAL_HOST_PLATFORM_NAME})

